sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/parser/ExpressionParserSuite.scala

```diff
@@ -653,6 +653,19 @@ class ExpressionParserSuite extends PlanTest {
       checkIntervals(s"'$value' day to second", result)
     }
 
+    // Day-Time intervals.
+    val hourTimeValues = Seq(
+      "11:22:33.123456789",
+      "11:22:33.123456789",
```

Comment:
Why do we have two same test coverage? It seems that this is copied from line 654~650 blindly.